KeePassRPC/KeePassRPCService.cs

```diff
@@ -1649,9 +1669,9 @@ private Group[] GetChildGroups(PwDatabase pwd, PwGroup group, bool complete, boo
                 {
                     kpg.ChildGroups = GetChildGroups(pwd, pwg, true, fullDetails);
                     if (fullDetails)
-                        kpg.ChildEntries = (Entry[])GetChildEntries(pwd, pwg, fullDetails);
+                        kpg.ChildEntries = (Entry[])GetChildEntries(pwd, pwg, fullDetails, false);
                     else
-                        kpg.ChildLightEntries = GetChildEntries(pwd, pwg, fullDetails);
+                    	kpg.ChildLightEntries = GetChildEntries(pwd, pwg, fullDetails, false);
```

Comment:
whitespace damage here and in a few other places in the diff (don't replace spaces with tabs - let's try to keep things consistent with current style)

Reply:
Oops, I'll blame SharpDevelop for that. I also found a lot of trailing whitespace that also got cleaned up as well.